src/sqs.rs

```diff
@@ -3,3 +3,44 @@
 #![cfg_attr(feature = "nightly-testing", allow(while_let_loop))]
 
 include!(concat!(env!("OUT_DIR"), "/sqs.rs"));
+
+#[cfg(test)]
+mod test {
+	use sqs::{SqsClient, CreateQueueRequest};
+    use super::super::{Region, SignedRequest};
+    use super::super::mock::*;
+
+    extern crate env_logger;
+
+    #[test]
+    // sample response from the SQS documentation
+    fn should_parse_example_create_queue_response() {
+        let mock = MockRequestDispatcher::with_status(200)
+            .with_body(r#"
+				<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe no indents on the string literal? It would make it a little easier to read.